src/Cake.NuGet/Install/Extensions/NuGetPackageManagerExtensions.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Frameworks;
+using NuGet.PackageManagement;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+using NuGet.ProjectManagement;
+using NuGet.Protocol.Core.Types;
+using NuGet.Resolver;
+
+namespace Cake.NuGet.Install.Extensions
+{
+    internal static class NuGetPackageManagerExtensions
+    {
+        public static async Task<IEnumerable<NuGetProjectAction>> GetInstallProjectActionsAsync(
+            this NuGetPackageManager packageManager,
+            NugetFolderProject nuGetProject,
+            PackageIdentity packageIdentity,
+            ResolutionContext resolutionContext,
+            INuGetProjectContext nuGetProjectContext,
+            IEnumerable<SourceRepository> sources,
+            CancellationToken token)
+        {
+            if (nuGetProject == null)
+            {
+                throw new ArgumentNullException(nameof(nuGetProject));
+            }
+
```

Comment:
Remove spaces between guard statements. My eyes hurt! 😄

Reply:
Ah, stupid Rider did stupid things because stupid me didn’t tell it that thus is stupid.